### DISCUSSION TITLE
Updated image in the Docker Hub

### DISCUSSION THREAD

**Discussion by linux-lukas** [pretalx/pretalx#1177] [General]
It would certainly be good if there was an up-to-date image in the Docker Hub for the latest stable pretalx version, or?

**Comment by rixx** [comment]
First off: There's no official docker image for pretalx. The existing docker image has always been community-maintained – you can find it at [pretalx/pretalx-docker](https://github.com/pretalx/pretalx-docker). When Docker moved automatic builds to their paid tier, our build integration stopped working. It would be great if somebody who regularly uses the pretalx docker image stepped up to maintain it.

Until then, I added a github-actions integration that should automatically push new releases to dockerhub in the future – until something else there changes or breaks. v2.2.0 is now on [hub.docker.com/pretalx/standalone](https://hub.docker.com/repository/docker/pretalx/standalone)

**Reply by linux-lukas** [reply]
Thank you for your quick reply! 

I will see what I can do to help in the future.

**Reply by linux-lukas** [reply]
Since I will probably have more to do with pretalx in the future, i.e. providing it for an honorary post, I would look to try and maintain it on a regular basis, @rixx
